src/PrimaryCensoredDist.jl

```diff
@@ -64,18 +64,23 @@ end
 Base.eltype(::Type{<:PrimaryCensoredDist{D}}) where {D} = promote_type(eltype(D), eltype(D))
 
 function Distributions.cdf(d::PrimaryCensoredDist, x::Real)
-    if x <= 0
-        return 0
+    solver = QuadGKJL()
```

Comment:
its not really clear to me what purpose defining this here has? I agree we want to define it elsewhere but why make this small change in this PR